interview-questions.md

```diff
@@ -8,36 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: You can fix this by migration. The name of the foreign key would be student_id, and it would be on the student model. This is because the cohort model already has an id that the many students would have to match up with.
 
-Researched answer:
+Researched answer: Migration can fix this issue. You will need to change the student model to include the foreign key, which in this case would be student_id. The cohort model already has a built in id that once you add the student id, because you already have it set to the cohort has_many students, it will automatically associate those things when you add the student id. Once you add that, you can migrate it and it will save the student model with the new student_id tag.
```

Comment:
When referencing a primary key from a different table, it is referred as a foreign key. The syntax is cohort_id:integer since the Student belongs to Cohort.
https://github.com/learn-academy-2022-echo/Syllabus/blob/main/rails/associations.md